process/block/alteredaccounts/alteredAccountsProvider_test.go

```diff
@@ -311,7 +370,12 @@ func testExtractAlteredAccountsFromPoolShouldIncludeESDT(t *testing.T) {
 	require.NoError(t, err)
 
 	encodedAddr := hex.EncodeToString([]byte("addr"))
```

Comment:
It looks quite confusing why the encoded address is the result of: `hex.EncodeToString`. It requires that you are aware that the pubKeyConverter uses hex package as encoder. If the Mock changes code inside, then this test will fail. Please use the following:
```suggestion
	encodedAddr := args.AddressConverter.Encode([]byte("addr"))
```

Reply:
you're right. changed